TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/human/HumanActivity.java

```diff
@@ -95,6 +81,12 @@ public void onConnect(String serverAddress) {
     };
 
     @Override
+    protected void onStart() {
+        super.onStart();
+        presenter = new HumanPresenter(SocketIOTpService.getInstance(), this);
```

Comment:
And if we don't pass `service` in the constructor, initialisation can still be done in `onCreate` and here just some sort of a setter. Or startPresenting with a `service` as a parameter ?